app/Request/Contract/SaveCarRequest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Request\Contract;
+
+use App\Entity\Car;
+use App\Entity\User;
+
+interface SaveCarRequest
+{
+    /**
+     * @return Car
+     */
+    public function getCar(): Car;
+
+    /**
+     * @return string|null
+     */
+    public function getColor();
```

Comment:
How about this approach `getColor(): ?string` ? Much cleaner and we can remove comments.